test/units/parsing/test_splitter.py

```diff
@@ -173,3 +173,9 @@ def test_split_args_error(args):
 def test_parse_kv_error(args):
     with pytest.raises(AnsibleParserError):
         parse_kv(args)
+
+
+@pytest.mark.parametrize("args", ('creates=something thisisextra', 'something thisisextra'))
```

Comment:
Keep the parameters to one-per-line so it's easier to read and add new ones:

```suggestion
@pytest.mark.parametrize("args", (
    'creates=something thisisextra', 
    'something thisisextra',
))
```